app/cypress/support/commands.js

```diff
@@ -349,16 +349,16 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add(
-  "addEmissionIntensityReport",
+  "addOrEditEmissionIntensityReport",
   (
     measurementPeriodStartDate,
     measurementPeriodEndDate,
     emissionFunctionalUnit,
     baselineEmissionIntensity,
     targetEmissionIntensity,
     postProjectEmissionIntensity,
-    productionFunctionalUnit = undefined,
-    adjustedEmissionsIntensityPerformance = undefined
+    productionFunctionalUnit,
+    adjustedEmissionsIntensityPerformance
   ) => {
```

Comment:
nit: I wonder about doing the arguments as an object? Then it wouldn't matter what order they were entered in. (I've entered them in the wrong order before and confused myself.) If you do make this change, I'd suggest only doing it for this new function--changing all the old ones would be a lot of work